docs/import/create_score_asset.js

```diff
@@ -305,26 +320,123 @@ function createScoreAsset(
       processing = processing.map(addTractInfo);
 
       processing = innerJoin(processing, sviPath, tractTag, cdcGeoidKey);
-      processing = processing.map((f) => combineWithAsset(f, sviTag, sviKey));
+      // Remove tract tag from final feature.
+      processing = processing.map(
+          (f) => combineWithAsset(f, sviTag, sviKey).set(tractTag, null));
+    }
+
+    if (buildingPath) {
+      // Get building count by block group.
+      const buildingsHisto =
+          computeBuildingsHisto(buildingPath, stateGroups, geoidTag);
+      processing = combineWithBuildings(processing, buildingsHisto);
     }
 
-    // Get building count by block group.
-    const buildingsHisto =
-        buildingPath ? computeBuildingsHisto(buildingPath, stateGroups) : null;
+    if (damage) {
+      // TODO(janakr): Assumes that damage asset does not include undamaged
+      //  buildings. Should probably offer option to filter.
+      processing = combineWithDamage(processing, damage);
+    }
 
-    // Create final feature collection.
-    const additionalTags = [];
-    if (incomePath) additionalTags.push(incomeTag);
-    if (sviPath) additionalTags.push(sviTag);
-    processing = processing.map(
-        (f) =>
-            countDamageAndBuildings(f, damage, buildingsHisto, additionalTags));
     allStatesProcessing = allStatesProcessing.merge(processing);
   }
 
   return backUpAssetAndStartTask(allStatesProcessing);
 }
 
+/**
+ * Performs operation of processing inputs and creating output asset for a
+ * disaster whose data is fairly flexible, with few assumptions. We expect:
+ * 1. A poverty asset;
```

Comment:
What do you think is the best way to document this flow? I think it's as clear here as text will allow but would be nice to have some kind of flow chart or something (maybe I'm just a visual learner) for this series of if else choices

Reply:
I think once the front-end is available, that will be the best documentation outside of the doc/code itself. It should walk the user through as they make selections, so the developer can also use it. I don't anticipate, for instance, making this flow explicit to the user. They should just choose the assets they have, and we'll give them the next choice based on that asset, with explanation.

Reply:
I think you're right the front end will be the best docs. Might be nice to replicate a flow chart into a doc in case we have front end bugs so it's never the full source of truth on the behavior we expect.